README.md

```
@@ -17,6 +17,12 @@ altairdsk allows you to:
 
 _Note: If you prefer the C version, you can find that under the "legacy" branch. I don't provide support for this version anymore._
 
+### LLM / "AI" Usage Disclosure ###
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)